spec/models/service_template_spec.rb

```diff
@@ -395,17 +395,17 @@
 
     it "should pass display attribute to created top level service" do
       @svc_a.display = true
-      expect(@svc_a.create_service(double(:options => {:dialog => {}}, :get_user => service_user)).display).to eq(true)
```

Comment:
Note, this is really confusing but the service template also has display but didn't get the visible column rename and doesn't have the deprecation.

Either way, I noticed this area of specs were raising these deprecations:

```
DEPRECATION WARNING: display is deprecated and will be removed from ManageIQ K-release (visible) (called from block (3 levels) in <top (required)> at /home/travis/build/jrafanie/manageiq/spec/models/service_template_spec.rb:398)
.DEPRECATION WARNING: display is deprecated and will be removed from ManageIQ K-release (visible) (called from block (3 levels) in <top (required)> at /home/travis/build/jrafanie/manageiq/spec/models/service_template_spec.rb:404)
........DEPRECATION WARNING: display is deprecated and will be removed from ManageIQ K-release (visible) (called from block (3 levels) in <top (required)> at /home/travis/build/jrafanie/manageiq/spec/models/service_template_spec.rb:408)
```